src/http/cookie.cr

```diff
@@ -8,30 +8,35 @@ module HTTP
     property value : String
     property path : String
     property expires : Time?
+    property max_age : Time::Span?
     property domain : String?
     property secure : Bool
     property http_only : Bool
     property extension : String?
+    @creation_time : Time
 
     def_equals_and_hash name, value, path, expires, domain, secure, http_only
 
     def initialize(@name : String, value : String, @path : String = "/",
-                   @expires : Time? = nil, @domain : String? = nil,
+                   @expires : Time? = nil, @max_age : Time::Span? = nil, @domain : String? = nil,
                    @secure : Bool = false, @http_only : Bool = false,
                    @extension : String? = nil)
+      @creation_time = Time.now
       @name = URI.unescape name
       @value = URI.unescape value
     end
 
     def to_set_cookie_header
       path = @path
       expires = @expires
+      max_age = @max_age
       domain = @domain
       String.build do |header|
         header << "#{URI.escape @name}=#{URI.escape value}"
         header << "; domain=#{domain}" if domain
         header << "; path=#{path}" if path
         header << "; expires=#{HTTP.rfc1123_date(expires)}" if expires
+        header << "; max-age=#{max_age.total_seconds}" if max_age
```

Comment:
I think `+` is preferred in string concatenation.
before:
```crystal
header << "; max-age=#{max_age.total_seconds}" if max_age
```

```crystal
header << "; max-age=" + max_age.total_seconds if max_age
```

Reply:
Seems like the rest of the headers are in the other style so this is not a change is not necessary with this PR.

Reply:
No, `+` is discouraged. Maybe `<<` would be slightly faster, but interpolation is totally fine.

Reply:
In that case this is good.